src/pages/Profile/index.js

```diff
@@ -23,90 +33,366 @@ import avatar from '../../assets/avatar.svg';
 class Profile extends React.Component {
   constructor(props) {
     super(props);
+    this.ref = {
+      name: React.createRef(),
+      email: React.createRef(),
+      position: React.createRef(),
+    };
     this.state = {
-      isChecked: true,
+      dataExpireValue: profileData.dataExpireOptions,
```

Comment:
I didn't notice this until re-reading Nikhil's comment below, but the mock data property is `dataExpireValue` and the property referenced here doesn't appear to exist in the `mock/user.js` file's `profileData`.

Also, it still seems to me that we should be loading the mock `companyID` into the state, even if we don't allow editing it; the "real code" won't be able to see it otherwise without a `REST` call for every render.